blade-image-build/ROOTFS/opt/sage/blade/image/base_packages

```diff
@@ -0,0 +1 @@
+ssh ansible git htop iotop iftop bwm-ng screen nmap sshfs autossh network-manager apt-transport-https ca-certificates curl gnupg-agent software-properties-common
```

Comment:
This file should be referenced in the pressed file in the 'late_command' to copy it from the dockerfs to the bladefs
*see last line in pressed file, where we copy other files it's just a cp you just need to make sure the path is what is right, and we have found no problems with putting files on /root but putting them in specific locations like /etc/ has caused files to be truncated sometimes.

Reply:
so basically add this `cp /cdrom/base_packages /target/root/base_packages` ??

Reply:
Okay i added it. thanks!